de4dot.code/deobfuscators/dotNET_Reactor/v4/EncryptedResource.cs

```diff
@@ -189,7 +189,7 @@ class DecrypterV1 : IDecrypter {
 			readonly byte[] key, iv;
 
 			public DnrDecrypterType DecrypterType {
-				get { return DnrDecrypterType.V1; }
+				get { return DnrDecrypterType.V1; } 
```

Comment:
You added a space at the end of the line. A tip is to diff your code against master and you'll catch errors like this.